Scripts/Hooks/scratchpad-hook.lua

```diff
@@ -328,6 +331,32 @@ local function loadScratchpad()
         self:insert("")
     end
 
+    local function setTitleBar(page)
+        local notice = ''
+        for i,j in pairs(pagesnotice) do
+            notice = notice .. ' | ' .. j
+        end
+
+        if window then
+            window:setText(page.name .. notice)
+        else
+            log('setTitleBar() window not defined yet')
+        end
+    end
+
+    local function setPagesnotice(extid, str)
+        if not extid then
+            log('setPagesnotice() fail extid nil')
+            return
+        end
+        if str then
+            pagesnotice[extid] = str
+        else
+            table.delete(pagesnotice, extid)
+        end
+        setTitleBar(pages[1])
```

Comment:
Is it correct that this always uses the first page to set the title bar?

Reply:
You are correct, this should have been the setting titlebar for currentPage as it includes the page name.

Reply:
titlebar handling removed